packages/trace-viewer/src/ui/errorsTab.tsx

```diff
@@ -72,6 +118,9 @@ export const ErrorsTab: React.FunctionComponent<{
           {location && <div className='action-location'>
             @ <span title={longLocation} onClick={() => revealInSource(error)}>{location}</span>
           </div>}
+          <span style={{ position: 'absolute', right: '5px' }}>
```

Comment:
5px is a very unusual value. Why is this not a multiple of 4?

Reply:
idk, the padding above is also 5px 🤷